RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -2,9 +2,12 @@
 
 #include "Event.h"
 
-using namespace ALPAKA_ACCELERATOR_NAMESPACE;
+using Device = ALPAKA_ACCELERATOR_NAMESPACE::Device;
+using Queue = ALPAKA_ACCELERATOR_NAMESPACE::Queue;
+using Acc1D = ALPAKA_ACCELERATOR_NAMESPACE::Acc1D;
+using Acc3D = ALPAKA_ACCELERATOR_NAMESPACE::Acc3D;
 
-void lst::Event<Acc3D>::initSync(bool verbose) {
+void ALPAKA_ACCELERATOR_NAMESPACE::lst::Event::initSync(bool verbose) {
```

Comment:
I don't see this pattern in other dev.cc files, `namespace ALPAKA_ACCELERATOR_NAMESPACE {` is used instead at the top of the file

Reply:
Okay, I switched to `using namespace ALPAKA_ACCELERATOR_NAMESPACE::lst;` at the top